app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,49 @@
+@import "foundation_and_overrides";
+
+body {
+  background: url("Home_photo.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+
+.footer {
+  background: transparent;
+  text-align: center;
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  height: 4rem;
+  padding: 0.5rem;
+}
+  .footer .links a {
+    border: 1px dotted white;
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.